js/languages/en-US.json

```diff
@@ -551,7 +553,7 @@
     "pageUnavailable": "This page is currently unavailable.",
     "badHandle": "The handle you entered does not have a valid OpenBazaar ID.",
     "serverDown": "The server has been shut down",
-    "unableToLoadImages": "We were unable to load your image(s).",
+    "unableToLoadImages": "We were unable to load your image.  |||| We were unable to load your images.",
```

Comment:
The smart count is passed in, and determines whether the first or second phrase should be used. It doesn't need to be in the actual phrase anywhere. The documentation isn't super clear on that. ;)

Hopefully the translators will know how to handle it, unfortunately we can't put comments into the language files. 

The extra spaces are a typo, I'll take them out.